cms/articles/models.py

```diff
@@ -91,14 +91,14 @@ class StatisticalArticlePage(BundledPageMixin, RoutablePageMixin, BasePage):  #
     parent_page_types: ClassVar[list[str]] = ["ArticleSeriesPage"]
     subpage_types: ClassVar[list[str]] = []
     template = "templates/pages/statistical_article_page.html"
-    label = _("Article")
+    label = "Article"
```

Comment:
```suggestion
    label = _("Article")
```

the label attribute is used on the FE